requirements.txt

```diff
@@ -11,8 +11,8 @@ git+https://github.com/dwf/picklable_itertools.git#egg=picklable-itertools
 
 --allow-external theano
 --allow-unverified theano
-git+https://github.com/Theano/Theano.git#egg=theano
+git+https://github.com/Theano/Theano.git@c6d2ab3929182cdf7cf12e4b1f191a6ff99d07cc#egg=theano
 
 --allow-external fuel
 --allow-unverified fuel
-git+https://github.com/mila-udem/fuel#egg=fuel
+git+https://github.com/mila-udem/fuel@stable#egg=fuel
```

Comment:
I merged it, but still, I guess, stable branches should be synchronized, and `@stable` is the right way to make a reference. But if when we have `0.0.2` or `0.1` or later versions, I think it is important to have a revision, let's say `old-release-0.0.1` that points to the right `Fuel` version and is installable.